src/components/layouts/Footer.jsx

```diff
@@ -0,0 +1,9 @@
+const Footer = () => {
+  return (
+    <footer className="flex w-screen  flex-row flex-wrap items-center justify-center bg-lightGreen w-full h-24 text-center text-black bottom-0">
```

Comment:
there are two width options w-screen and w-full. Please delete one of them

Reply:
deleted